spec/models/ext_management_system_spec.rb

```diff
@@ -258,7 +258,20 @@
   context "with virtual totals" do
     before(:each) do
       @ems = FactoryGirl.create(:ems_vmware)
-      (1..2).each { |i| FactoryGirl.create(:vm_vmware, :ext_management_system => @ems, :name => "vm_#{i}") }
+      (1..2).each { |i| FactoryGirl.create(:vm_vmware,
```

Comment:
looking good

please do not change, but for future reference: `2.times { FactoryGirl.create`